src/siteconfig/models.py

```diff
@@ -246,6 +246,11 @@ class SiteConfig(models.Model):
             or \"Skill\" might be a more suitable name, depending on how you use the Tags feature."
     )
 
+    custom_profile_field = models.CharField(
+        null=True, blank=True, max_length=20,
+        help_text="If left empty wont show up in user profiles"
```

Comment:
Use full sentences and proper grammar in these front facing text.
"This field can be used to collect custom data such as a student's grad year or other important info you need from your users.  If left blank this field will be ignored."